### PR TITLE
Fixes Cell Message With Mulebots

### PR DESCRIPTION
**Pull request by Wisemonster**
## About The Pull Request
Fixes the message when removing/adding a cell to mulebots using the mulebot as the user, resulting in sentient mulebots receiving the message "You add/pry out the cell in the mulebot" despite not being the one adding/removing the cell. First time making a pr for this codebase, so let me know if I fucked something up.
## Why It's Good For The Game
The mulebot isn't moving the cell, whoever IS moving it is. Also bug bad.
## Changelog
:cl:
fix: Fix mulebots receiving the wrong message when thier cell is added/removed.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/bot/mulebot.dm [hunk 1]
```diff
@@ -157,7 +157,7 @@
 		user.put_in_hands(cell)
 	else
 		cell.forceMove(drop_location())
-	visible_message(span_notice("[user] crowbars [cell] out from [src]."),
+	user.visible_message(span_notice("[user] crowbars [cell] out from [src]."),
 					span_notice("You pry [cell] out of [src]."))
```
Comment (san7890) [inline code/modules/mob/living/simple_animal/bot/mulebot.dm:161]:
```suggestion
	user.visible_message(
		span_notice("[user] crowbars [cell] out from [src]."),
		span_notice("You pry [cell] out of [src]."),
	)
```

File: code/modules/mob/living/simple_animal/bot/mulebot.dm [hunk 2]
```diff
@@ -172,7 +172,7 @@
 			return TRUE
 		cell = I
 		diag_hud_set_mulebotcell()
-		visible_message(span_notice("[user] inserts \a [cell] into [src]."),
+		user.visible_message(span_notice("[user] inserts \a [cell] into [src]."),
 						span_notice("You insert [cell] into [src]."))
```
Comment (san7890) [inline code/modules/mob/living/simple_animal/bot/mulebot.dm:176]:
```suggestion
		user.visible_message(
			span_notice("[user] inserts \a [cell] into [src]."),
			span_notice("You insert [cell] into [src]."),
		)
```

**Review by san7890 — CHANGES_REQUESTED** [review]
thanks for fixing this, there's one thing that i'd really appreciate if you commited while in the area, and that's just multilining the lines to look way way cleaner because otherwise it'll never get done by anyone

otherwise sounds great

**Comment by Wisemonster** [comment]
Alright, done

**Pull request merged**